frontend/src/lib/doc-util.tsx

```diff
@@ -51,16 +51,19 @@ export const DOC_LINKS = {
   MANAGE_APPLICATIONS: `${DOC_BASE_PATH}/applications/index#managing-applications`,
   ANSIBLE_JOBS: `${DOC_BASE_PATH}/clusters/index#ansible-config-cluster`,
   POLICIES_OVERVIEW: `${DOC_BASE_PATH}/governance/index#policy-overview`,
```

Comment:
Based on my comment here: https://issues.redhat.com/browse/ACM-15043
```suggestion
  POLICIES_OVERVIEW: `${DOC_BASE_PATH}/governance/index#hub-policy-framework`,
```

Reply:
Can this also be updated for 2.12?